cupy/core/core.pyx

```diff
@@ -1644,8 +1644,30 @@ cdef class ndarray:
                 a_gpu = asfortranarray(self)
             else:
                 raise ValueError("unsupported order: {}".format(order))
-        a_cpu = numpy.empty(self._shape, dtype=self.dtype, order=order)
+
+        if arr is not None:
+            if not isinstance(arr, numpy.ndarray):
+                raise TypeError('Only numpy.ndarray can be got from'
```

Comment:
Just a wording suggestion. "be got" is barely used.
Reference: https://ell.stackexchange.com/questions/113772/is-can-be-got-gotten-correct

```suggestion
                raise TypeError('Only numpy.ndarray can be obtained from'
```